Gems/PhysX/Code/Source/Material/PhysXMaterial.cpp

```diff
@@ -236,7 +269,12 @@ namespace PhysX
             "PhysX Material", restitution >= 0.0f && restitution <= 1.0f, "Restitution value %f will be clamped into range [0, 1]",
             restitution);
 
-        m_pxMaterial->setRestitution(AZ::GetClamp(restitution, 0.0f, 1.0f));
+        m_restitution = AZ::GetClamp(restitution, 0.0f, 1.0f);
+
+        if (!IsCompliantContactModeEnabled())
```

Comment:
suggestion: A comment here to explain why we don't set the restitution of compliant contacts are enabled would be super helpful

Reply:
done